code/modules/admin/admin_verbs.dm

```diff
@@ -148,6 +148,7 @@ var/list/admin_verbs_spawn = list(
 	/datum/admins/proc/spawn_plant,
 	/datum/admins/proc/spawn_atom,		//allows us to spawn instances,
 	/client/proc/respawn_character,
+	/client/proc/spawn_character_mob,

```

Comment:
Needs //VOREStation Edit,

The comma at the end is important to stop travis from freaking out.